corehq/apps/app_manager/models.py

```diff
@@ -5083,17 +5088,23 @@ def odk_profile_url(self):
     def odk_media_profile_url(self):
         return reverse('download_odk_media_profile', args=[self.domain, self._id])
 
-    def get_odk_qr_code(self, with_media=False, build_profile_id=None):
+    def get_odk_qr_code(self, with_media=False, build_profile_id=None, download_target_version=False):
         """Returns a QR code, as a PNG to install on CC-ODK"""
+        filename = 'qrcode.png' if not download_target_version else 'qrcode-targeted.png'
```

Comment:
What's the significance of the different filename? Should it include the specific target name?

Reply:
We need to use two different filenames for the targeted and non-targeted qrcodes, because we save these as attachments and need to distinguish.  The link doesn't specify which target version, just whether or not the download version is targeted.